### DISCUSSION TITLE
[Feature Request] Prevent opening previously read books in series from reader, resetting their read status/date

### DISCUSSION THREAD

**Discussion by Hattori0** [gotson/komga#1782] [Ideas]
#### **Is your feature request related to a problem? Please describe.**
Let's say I have comics collected under a series named after an artist. The comics can be read in any order, so continuity doesn't matter. Now, in a situation, where a couple of books in a series have been read already, it's possible to accidentally click too much at the last page in reader and automatically open a book that has already been read, resetting its read status and read date. 
I acknowledge that there is a brief popup warning against that already.

#### **Describe the solution you'd like**
Ideally, it should be possible to exclude read books from the pool of books that can be automatically opened after the last page of the currently open book, so that their status doesn't reset
or possibly add a toggle to disable moving to the next book in series by clicking on the side of the screen or perhaps just close the reader after the last page, as the read status and date can't be restored to their previous state, should they reset in this way.
Maybe another solution would be more viable in this case.

#### Closing
To the best of my knowledge this particular issue hasn't been opened in an issue before or otherwise suggested.
At the time of writing, I'm using the latest version of Komga on windows **v1.14.1-master**, from the portable archive.

Thank you

**Comment by gotson** [comment]
You should not use series for that kind of use case, but one shots.
